scripts/pico_project.py

```diff
@@ -578,69 +568,35 @@ def GenerateCMake(folder, params):
                  "# Initialise pico_sdk from installed location\n"
                  "# (note this can come from environment, CMake cache etc)\n\n"
                 )
-    
+
     # if you change the do never edit headline you need to change the check for it in extension.mts
-    cmake_header_us_1_5_1 = (
-                "# == DO NEVER EDIT THE NEXT LINES for Raspberry Pi Pico VS Code Extension to work ==\n"
-                "if(WIN32)\n"
-                "   set(USERHOME $ENV{USERPROFILE})\n"
-                "else()\n"
-                "    set(USERHOME $ENV{HOME})\n"
-                "endif()\n"
-                f"set(PICO_SDK_PATH {cmakeSdkPath(params['sdkVersion'])})\n"
-                f"set(PICO_TOOLCHAIN_PATH {cmakeToolchainPath(params['toolchainVersion'])})\n"
-                "if(WIN32)\n"
-                f"    set(pico-sdk-tools_DIR {cmakeToolsPath(params['sdkVersion'])})\n"
-                "    include(${pico-sdk-tools_DIR}/pico-sdk-tools-config.cmake)\n"
-                "    include(${pico-sdk-tools_DIR}/pico-sdk-tools-config-version.cmake)\n"
-                "endif()\n"
-                "# ====================================================================================\n"
-                )
     cmake_header_us = (
                 "# == DO NEVER EDIT THE NEXT LINES for Raspberry Pi Pico VS Code Extension to work ==\n"
                 "if(WIN32)\n"
-                "   set(USERHOME $ENV{USERPROFILE})\n"
+                "    set(USERHOME $ENV{USERPROFILE})\n"
                 "else()\n"
                 "    set(USERHOME $ENV{HOME})\n"
                 "endif()\n"
-                f"set(PICO_SDK_PATH {cmakeSdkPath(params['sdkVersion'])})\n"
-                f"set(PICO_TOOLCHAIN_PATH {cmakeToolchainPath(params['toolchainVersion'])})\n"
-                f"set(pioasm_HINT {cmakeToolsPath(params['sdkVersion'])}/pioasm)\n"
-                "if(EXISTS ${pioasm_HINT})\n"
-                "    set(pioasm_DIR ${pioasm_HINT})\n"
-                "endif()\n"
-                f"set(picotool_HINT {cmakePicotoolPath(params['picotoolVersion'])}/picotool)\n"
-                "if(EXISTS ${picotool_HINT})\n"
-                "    set(picotool_DIR ${picotool_HINT})\n"
-                "endif()\n"
-                "if(PICO_TOOLCHAIN_PATH MATCHES \"RISCV\")\n"
-                "    set(PICO_PLATFORM rp2350-riscv CACHE STRING \"Pico Platform\")\n"
-                "    if(PICO_TOOLCHAIN_PATH MATCHES \"COREV\")\n"
-                "        set(PICO_COMPILER pico_riscv_gcc_zcb_zcmp)\n"
-                "    endif()\n"
-                "endif()\n"
+                f"set(sdkVersion {params['sdkVersion']})\n"
+                f"set(toolchainVersion {params['toolchainVersion']})\n"
+                f"set(picotoolVersion {params['picotoolVersion']})\n"
```

Comment:
Great, this also directly fixes an issue I noticed during testing that currently the extension always generates a `CMakeLists.txt` with a picoTool version equal to it's selected SDK version (see L593).